From 45ba681f5f1a27f0b0b87bf961bce014fe73e056 Mon Sep 17 00:00:00 2001
From: Pablo Ceballos <pceballos@google.com>
Date: Thu, 26 May 2022 22:23:35 +0000
Subject: [PATCH] hwdb: Add Google Meet speakermic

It has the same issue as the Google Hangouts Meet speakermic but a
different VID/PID.

(cherry picked from commit 1a2134fb51ab1e84c213f2a309dab25da3a5369d)

Related: #2087778
---
 hwdb.d/60-keyboard.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 2e4f6f4293..7204e4ea2a 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -515,8 +515,9 @@ evdev:input:b0003v0458p0708*
 # Google
 ###########################################################
 
-# Google Hangouts Meet speakermic
+# Google Hangouts Meet speakermic and Google Meet speakermic
 evdev:input:b0003v18D1p8001*
+evdev:input:b0003v18D1p8007*
  KEYBOARD_KEY_b002f=reserved                            # Disable micmute key
 
 ###########################################################
